sympy/ntheory/primetest.py

```diff
@@ -110,6 +110,24 @@ def mr(n, bases):
     return True
 
 
+def isprimeE(n, b):
+    """isprimeE(n ,b) - Test whether n is prime or an Euler pseudoprime to base b."""
+	if (not mr(n, b)): return False
```

Comment:
`mr` expects a list of bases so this should be passed as `mr(n, [b])`.

Reply:
Sorry for this silly typo.
Its fixed now thanks!